routes/originGitHub.js

```diff
@@ -20,13 +21,30 @@ router.get(
       const { login, repo } = request.params
       const github = getClient(request)
       const answer = await github.gitdata.getTags({ owner: login, repo, per_page: 100 })
-      const result = answer.data
-        .map(item => {
-          return { tag: item.ref.slice(10), sha: item.object.sha }
-        })
+      const unsorted = await Promise.all(answer.data
+        .map(throat(5, async item => {
+          if (item.object.type === 'commit')
+            return { tag: item.ref.slice(10), sha: item.object.sha }
+
+          if (item.object.type !== 'tag') {
+            return null
+          }
+
+          // we know now that we have an annotated tag, get the associated commit hash
```

Comment:
can you point to an example of an annotated tag on a github repo? I'm having trouble testing this out

Reply:
Sure. The tag v2.4.1 (https://github.com/chalk/chalk/tree/v2.4.1) points to commit https://github.com/chalk/chalk/commit/48ba5b0b9beadcabd9fc406ac4d9337d8fa6b36d, but the annotated tag hash is e5b8eb4ae59a954938014c06bb7ec428235e06dc (which is the one you currently see in clearlydefined in the harvest page).

Reply:
I see. Makes sense. Thanks for the explanation!